t/ui/12-needle-edit.t

```diff
@@ -195,14 +194,21 @@ sub overwrite_needle($) {
     is($driver->find_element('#needleeditor_name', 'css')->get_value(), "", "needle name input clean up");
     $driver->find_element('#needleeditor_name', 'css')->send_keys($needlename);
     is($driver->find_element('#needleeditor_name', 'css')->get_value(), "$needlename", "new needle name inputed");
-    $driver->find_element('//input[@alt="Save"]')->click();
+    $driver->find_element('#save', 'css')->click();
     t::ui::PhantomTest::wait_for_ajax;
-    # check the state highlight changed and click Yes do overwrite then
-    is($driver->find_element('ui-state-highlight', 'class')->get_text(), "Same needle name file already exists! Overwrite it? Yes / No", "highlight appears correct");
-    $driver->find_element('Yes', 'link_text')->click();
+
+    #t::ui::PhantomTest::make_screenshot('mojoResults.png');
+    #print $driver->get_page_source();
```

Comment:
maybe you want to use [smart comments](http://search.cpan.org/~neilb/Smart-Comments-1.06/lib/Smart/Comments.pm)? I am used to and had good experiences with coding styles which don't allow disabled code unless it is very good explained.